cpg_pipes/config-template.toml

```
@@ -0,0 +1,254 @@
+# the --skip-samples-with-missing-input option controls whether such
+# should be ignored, or raise an error
+check_inputs = true
+
+# Within jobs, check all in-job intermediate files for possible reuse.
+# If set to False, will overwrite all intermediates
+check_intermediates = true
+
+# Before running a stage, check if its input already exists. If it exists, 
+# submit a [reuse] job instead.
+check_expected_outputs = true
+
+# Local directory for temporary files. Usually takes a few kB.
+# If not provided, a temp folder will be created
+#local_tmp_dir = 
+
+# Slack channel to send status reports with the CPG status reporter:
+#slack_channel = 'seqr-loader'
+
+# When the sample-metadata database API returns an error from the
+# database, only show the error and continue, instead of crashing
+smdb_errors_are_fatal = true
+
+# Prefix to find docker images referenced in `image_config_yaml_path`
+image_registry_prefix = 'australia-southeast1-docker.pkg.dev/cpg-common/images'
+
+# Prefix to find reference files referenced in `refdata_yaml_path`
+reference_prefix = 'gs://cpg-reference'
+
+# Template to build HTTP URLs matching the dataset_path of category 
+# "web". Should be parametrised by namespace and dataset in Jinja format:
+web_url_template = 'https://{namespace}-web.populationgenomics.org.au/{dataset}'
+
+# GCP project to activate and for requester-pays buckets. For CPG, will be determined
+# automatically if not provided, and analysis-runner service account is activated.
+#dataset_gcp_project =
+
+# Limit to data of this sequencing type
+sequencing_type = 'genome'
+
+# Realign CRAM when available, instead of using FASTQ.
+# The parameter value should correspond to CRAM version
+# (e.g. v0 in gs://cpg-fewgenomes-main/cram/v0/CPG01234.cram
+realign_from_cram_version = 'bwamem'
+
+# Number of shards for realignment from BAM/CRAM
+realignment_shards_num = 10
+
+# Number of intervals to partition sample genotyping
+hc_intervals_num = 50
+
+# Number of intervals to partition joint calling and VQSR
+jc_intervals_num = 50
+
+# Number of intervals to partition VEP annotation
+vep_intervals_num = 50
+
+# Use GnarlyGenotyper instead of GenotypeGVCFs
+use_gnarly = false
+
+# Use allele-specific annotations for VQSR
+use_as_vqsr = true
+
+# Run CRAM QC and PED checks
+cram_qc = true
+
+# Calling intervals (defauls to whole genome intervals)
+#intervals_path = 
+
+# Create Seqr ElasticSearch indices for these datasets. If not specified, will
+# create indices for all input datasets.
+#create_es_index_for_datasets = 
+
+[elasticsearch]
+# Configure access to ElasticSearch server
+es_port = 9243
+es_host = 'elasticsearch.es.australia-southeast1.gcp.elastic-cloud.com'
+es_username = 'seqr'
+#es_password = 
+# Load ElasticSearch password from a secret, unless SEQR_ES_PASSWORD is set
+es_password_secret_id = 'seqr-es-password'
+es_password_project_id = 'seqr-308602'
+
+[hail]
+#hail_billing_project = 
+#hail_bucket = 
+hail_pool_label = 'seqr'
```

It's already in the `hail` section:
```suggestion
pool_label = 'seqr'
```